sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/csv/CSVInferSchemaSuite.scala

```diff
@@ -252,6 +252,9 @@ class CSVInferSchemaSuite extends SparkFunSuite with SQLHelper {
     withSQLConf(SQLConf.TIMESTAMP_TYPE.key -> "TIMESTAMP_NTZ") {
```

Comment:
shall we remove this test? TIMESTAMP_TYPE config does not affect schema inference anymore.

Reply:
Yeah I am keeping it on purpose. It doesn't hurt to check the impact of the TIMESTAMP_TYPE on the behavior of CSV data source.